bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/Wprimetotb/Wprimetotb_LH_run_card.dat

```diff
@@ -0,0 +1,269 @@
+#                       MadGraph5_aMC@NLO                            *
+#                                                                    *
+#                     run_card.dat MadEvent                          *
+#                                                                    *
+#  This file is used to set the parameters of the run.               *
+#                                                                    *
+#  Some notation/conventions:                                        *
+#                                                                    *
+#   Lines starting with a '# ' are info or comments                  *
+#                                                                    *
+#   mind the format:   value    = variable     ! comment             *
+#                                                                    *
+#   To display more options, you can type the command:               *
+#      update full_run_card                                          *
+#*********************************************************************
+#
+#*******************                                                 
+# Running parameters
+#*******************                                                 
+#                                                                    
+#*********************************************************************
+# Tag name for the run (one word)                                    *
+#*********************************************************************
+  tag_1     = run_tag ! name of the run 
+#*********************************************************************
+# Number of events and rnd seed                                      *
+# Warning: Do not generate more than 1M events in a single run       *
+#*********************************************************************
+  10000 = nevents ! Number of unweighted events requested 
+  0   = iseed   ! rnd seed (0=assigned automatically=default))
+#*********************************************************************
+# Collider type and energy                                           *
+# lpp: 0=No PDF, 1=proton, -1=antiproton, 2=photon from proton,      *
+#                                         3=photon from electron     *
+#*********************************************************************
+     1        = lpp1    ! beam 1 type 
+     1        = lpp2    ! beam 2 type
+     6500.0     = ebeam1  ! beam 1 total energy in GeV
+     6500.0     = ebeam2  ! beam 2 total energy in GeV
+# To see polarised beam options: type "update beam_pol"
+#*********************************************************************
+# PDF CHOICE: this automatically fixes also alpha_s and its evol.    *
+#*********************************************************************
+     nn23lo1    = pdlabel     ! PDF set                                     
+     230000    = lhaid     ! if pdlabel=lhapdf, this is the lhapdf number
```

Comment:
Is there a reason to use NNPDF23 instead of NNPDF31?

Reply:
We already use this set for the RH production, already approved. We need an LO set and we already checked that this not cause any serious issue.
[Here](https://github.com/cms-sw/genproductions/blob/master/bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/Wprimetotb/Wprimetotb_run_card.dat) the link to the run card for the RH production.

Reply:
lhapdf    = pdlabel     ! PDF set                                     
     315200    = lhaid     ! if pdlabel=lhapdf, this is the lhapdf number
     $DEFAULT_PDF_SETS = lhaid
     $DEFAULT_PDF_MEMBERS  = reweight_PDF     ! reweight to get PDF uncertainty

Actually this card is not quite right. Guess 315200 = lhaid would be overwritten by the line below,  $DEFAULT_PDF_SETS = lhaid. But anyways unlike what you said you did not use the same set for RH production based on the cards you linked. lhaid 3XXXXX is NNPDF31, not NNPDF23

Reply:
What do you mean by "this set"? For the samples you already produced in the past used different PDF set and you ONLY want to change the PDF to LO? 
And you'd be using CP1-5 tunes for pythia showerings right? CP1-5 tunes is made to complement MC-data disagreements when using NNPDF31. So I think NNPDF23 will not be a reasonable choice.

Reply:
So you had RH production in the past, and now you want LH production, is this correct?
But would it not be better to synchronize the PDF settings with RH then? The PDF set used for [1] is clearly different, it's not NNPDF23

[1] https://github.com/cms-sw/genproductions/blob/master/bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/Wprimetotb/Wprimetotb_run_card.dat#L47-L50

Reply:
You are right indeed. The PDF used in the past was the 315200. In the previous cards the line 49 has been commented out when producing the GPs, but it was never updated here. I will update this card